bin/build

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+yamlFile="$(ls intersight-openapi-*.yaml)"
+gemVersion="$(grep VERSION lib/intersight_client/version.rb | cut -d "'" -f 2)"
+additionalProperties="legacyDiscriminatorBehavior=false,gemAuthor=XLAB,gemHomepage=https://github.com/xlab-si/intersight-sdk-ruby,gemDescription=\"Gem for interaction with Cisco Intersight API\",gemName=intersight_client,moduleName=IntersightClient,gemLicense=\"Apache-2.0\",gemVersion=$gemVersion"
+templateDir='bin/oas_generator/templates/'
+
+set -ex
+
+if [ -z "$SKIP_GENERATE" ]; then
+	if [ -n "$LOCAL_CLI" ]; then
```

Comment:
SKIP_GENERATE was added because the actual generate can take hours. This is mostly used for development work on the patches where I can generate once, commit that, then skip the generate and run the "rest" of the patching, reverting back to the committed version if something goes wrong.

LOCAL_CLI was added because, at least on Mac, docker IO is so incredibly slow that the build took almost 3 hours...LOCAL_CLI was closed to 1 hour.